R/function-filtering.R

```diff
@@ -0,0 +1,147 @@
+#' @title Basic filtering functions for metabolomics
+#'
+#' @description
+#'
+#' When dealing with metabolomics results, it is often necessary to filter
+#' features based on certain criteria. These criteria are typically derived
+#' from statistical formulas applied to full rows of data, where each row
+#' represents a feature. The following functions provide basic filtering
+#' methods commonly used in the analysis of metabolomics data.
+#'
+#' - `rsd` and `rowRsd` are convenience functions to calculate the relative
+#'  standard deviation (i.e. coefficient of variation) of a numerical vector
+#'  or for rows of a numerical matrix, respectively.
+#'
+#' - `rowDratio` computes the D-ratio or "dispersion ratio," defined as the
+#'  standard deviation for QC (Quality Control) samples divided by the
+#'  standard deviation for biological test samples, for each feature (row) in
+#'  the matrix.
+#'
+#' - `percentMissing` and `rowPercentMissing` determine the percentage of
+#'  missing values in a vector or for each row of a matrix, respectively.
+#'
+#' - `rowBlank` identifies rows (i.e features) where the mean of test samples
+#'  is greater than twice the mean of blank samples. This can highlights
+#'  features that results from contamination in the solvent of the samples.
+#'   Return a `logical` vector
```

Comment:
Maybe rather *Returns a `logical` vector of length equal to the number of rows of `x`.*